src/opsgenie_alert_read.html

```diff
@@ -0,0 +1,33 @@
+<script type="text/javascript">
+    RED.nodes.registerType('OpsgenieAlertRead',{
+        category: 'Opsgenie',
+        color: '#2684ff',
+        defaults: {
+            parameters: {type:"opsgenie-config", required:true},
+            name: {value:""}
+        },
+        inputs:1,
+        outputs:1,
+        paletteLabel: "Opsgenie Alert Read",
+        icon: "font-awesome/fa-bell",

```

Comment:
Let's keep all the alert nodes the same in terms of icon
```suggestion
        icon: "alert.png",

```